json-webhook/functions/event-to-webhook.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require('node-fetch');
+
+exports.handler = async function(context, event, callback) {
+  try {
+    // IFTTT only pulls the fields value1, value2, and value3 from webhook JSON;
+    event.value1 = event.MessageSid;
+    event.value2 = event.From;
+    event.value3 = event.Body;
+
+    const res = await fetch(context.WEBHOOK_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(event),
```

Comment:
by adding in `value1/2/3` to the `event`. Wouldn't this make the payload have duplicated data with the object containing something like
```
{
  value1: "MS123",
  MessageSid: "MS123",
  value2: "+1555",
  From: "+1555",
  value3: "Hey there",
  Body: "Hey there",
}
```

will this lead to confusing results in customer's webhooks using this?

Reply:
It shouldn't be too confusing in Zapier's UI, as it has full-text search for fields based on the object their webhook received during Zap creation. IFTTT only passes `value1`, `value2`, and `value3` to its workflows, so the function will need a slight edit for customers who want other data passed to IFTTT, which we may be able to make slightly more clear in the code.

If desired, I may be able to make the function switch the `value1`, `value2`, and `value3` fields on and off based on a flag. Do our `.env` fields have anything like a boolean type that could render as an "IFTTT Compatibility" checkbox?

Reply:
Cool, I'm ok leaving this as is for now